### PR TITLE
APIS-5320 - Upgrade auth client lib thru boostrap play

### PR DESCRIPTION
**Pull request by AndySpaven**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -17,13 +19,12 @@ lazy val scoverageSettings = {
...
 def compileDeps = Seq(
-  "uk.gov.hmrc" %% "bootstrap-play-26" % "2.1.0",
+  "uk.gov.hmrc" %% "bootstrap-play-26" % "4.0.0",
   "uk.gov.hmrc" %% "auth-client" % "3.2.0-play-26",
   "com.beachape" %% "enumeratum-play-json" % "1.6.0")
 
 def testDeps(scope: String) = Seq(
-  "uk.gov.hmrc" %% "hmrctest" % "3.9.0-play-26" % scope,
-  "org.scalatest" %% "scalatest" % "3.0.8" % scope,
+  // "uk.gov.hmrc" %% "hmrctest" % "3.9.0-play-26" % scope,
```
Comment (AndySpaven) [inline build.sbt]: remove this

File: .gitignore [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 !.gitignore
 !.scalafmt.conf
+!.scalafix.conf
```
Comment (mattgreenuk) [inline .gitignore:5]: Should we have done scalafix changes on a separate PR?

**Review by mattgreenuk — APPROVED** [review]
